lms/static/scripts/frontend_apps/components/dashboard/PaginatedMultiSelect.tsx

```diff
@@ -96,7 +96,7 @@ export default function PaginatedMultiSelect<TResult, TSelect>({
       value={value}
       onChange={onChange}
       aria-label={`Select ${entity}`}
-      containerClasses="!w-auto min-w-[180px]"
+      containerClasses="!w-auto min-w-44"
```

Comment:
The tailwind version we were using when this was implemented didn't support sizes for `min-w-*`, so we hardcoded the size in px.

We now use a version that supports it. In this case, min-w-44 is 176px, which is close.